components/Calendars.js

```diff
@@ -257,6 +259,248 @@ class RenderAgendaBase extends Component {
 
 export const RenderAgenda = withNavigation(withContext(RenderAgendaBase))
 
+class ReservationAgendaBase extends Component {
```

Comment:
I think agenda may not be the right word here. Let's change to ReservationCalendarBase.